From 250a3a7cee1a4bbfc432ab6da5b986e889207526 Mon Sep 17 00:00:00 2001
From: Aaron Plattner <aplattner@nvidia.com>
Date: Wed, 10 Mar 2010 18:31:08 -0800
Subject: [PATCH 3/3] Update MCP6x supported products.

Use the official names for 10DE:053B and 10DE:053E.  Add some additional
product IDs and update the documentation.

Signed-off-by: Aaron Plattner <aplattner@nvidia.com>
---
 man/nv.man      |    2 +-
 src/nv_driver.c |    9 ++++++---
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/man/nv.man b/man/nv.man
index d7d8ae9..228f1da 100644
--- a/man/nv.man
+++ b/man/nv.man
@@ -57,7 +57,7 @@ NV30, NV31, NV34, NV35, NV36, NV37, NV38
 NV40, NV41, NV43, NV44, NV45, C51
 .TP 22
 .B GeForce 7XXX
-G70, G71, G72, G73
+G70, G71, G72, G73, MCP6x
 .TP 22
 .B GeForce 8XXX, GeForce 9XXX, nForce 7, GeForce G, ION
 G8x, G9x, GT21x, MCP7x
diff --git a/src/nv_driver.c b/src/nv_driver.c
index 99d172f..0d017f9 100644
--- a/src/nv_driver.c
+++ b/src/nv_driver.c
@@ -125,7 +125,7 @@ _X_EXPORT DriverRec NV = {
 #endif
 };
 
-/* Known cards as of 2010/03/08 */
+/* Known cards as of 2010/03/10 */
 
 static SymTabRec NVKnownChipsets[] =
 {
@@ -380,8 +380,11 @@ static SymTabRec NVKnownChipsets[] =
   { 0x10DE03D2, "GeForce 6100 nForce 400" },
   { 0x10DE03D5, "GeForce 6100 nForce 420" },
 
-  { 0x10DE053E, "GeForce 7025" },
-  { 0x10DE053B, "GeForce 7050" },
+  { 0x10DE0531, "GeForce 7150M / nForce 630M" },
+  { 0x10DE0533, "GeForce 7000M / nForce 610M" },
+  { 0x10DE053A, "GeForce 7050 PV / nForce 630a" },
+  { 0x10DE053B, "GeForce 7050 PV / nForce 630a" },
+  { 0x10DE053E, "GeForce 7025 / nForce 630a" },
 
 /*************** G8x ***************/
   { 0x10DE0191, "GeForce 8800 GTX" },
-- 
1.7.0.2

